config/initializers/sample_manifest_excel.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-Rails.application.config.to_prepare do
+Rails.application.config.after_initialize do
```

Comment:
A comment would be good to explain config.after_initialize is to run once after the app is initialized.